pkg/framework/aries/framework.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package aries
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/transport"
+	errors "golang.org/x/xerrors"
+)
+
+// Framework provides access to clients being managed by the Framework.
+type Framework struct {
```

Comment:
I would have thought the aries package's New would return a *Aries?

Also the comment is a bit hard to grok (Framework ... Framework): // Framework provides access to clients being managed by the Framework.

Reply:
It would stutter if we have `Aries` as the name, as we already have package name as `aries`.

Reply:
This isn't stutter.

Reply:
It's normal for a package that supplies/constructs a single type to have the same name as the struct.

For an explanation, I like the first answer here: https://stackoverflow.com/questions/45885985/ways-to-avoid-stuttering-in-go-package-and-struct-names

> What you want to avoid is unnecessary stuttering

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/b6b4c89d93e1cd85b02b49cf33b3cd03858e5bba..e7224c2166bc28f01e32148341d7f003d8fe83c0#diff-7b2448fdc2181299533bc944239b9744R15).